src/Faker/Provider/ms_MY/Company.php

```diff
@@ -0,0 +1,105 @@
+        'Racun Berjadual','Racun Serangga','Radar Dan Alatganti','Rakaman','Rawatan Hutan','Reaktor dan Instrumen Nuklear','Rekabentuk Percetakan','Renting','Resort','Roket Dan Sub Sistem, Pelancar','Rotan','Ruang Niaga','Rumah Kediaman','Rumah Tumpangan',
+        'Salvage Boat','Sampan','Sampel dan Sampel Awetan Haiwan','Sand Blasting Dan Mengecat Untuk Kapal','Satelit','Satelit Dan Alatganti','Semua Peralatan Sukatan','Senjata Api','Serangga','Sesalur','Shelf packages including maintenance','Ship Chandling','Ship Trimming','Simulator','Simulator Bot','Simulator serta lain-lain','Sisa Perawatan','Sistem Elektrik','Sistem Elektronik','Sistem Pencegah Kebakaran','Sistem Perhubungan','Sistem, Peralatan, Alat Ganti Keretapi Dan Aksesori','Software','Solekan','Split','Stesen Janakuasa, Peralatan','Stevedor','Stor','Sub Sistem Roket','Sukan','Sumber Air','Sungai','Syarikat Insuran','Syarikat pelelong awam','System development',
+        'Tag','Talian Paip','Taman','Tanaman','Tanda Dan Stiker','Tangki','Tasik','Tatahias Haiwan','Teknologi Hijau','Teknologi Maklumat Dan Komunikasi','Tekstil','Tekstil Guna Semula Kakitangan','Tekstil Pakai Buang Kakitangan','Telecommunication','Telekomunikasi','Telly Clerk','Tempat Letak Kereta','Tenaga Buruh','Ternakan','Terusan','Topografi','Trailer Dan Aksesori','Tukun Tiruan','Tumbuhan',
+        'Ubat Haiwan','Ubat Tidak Berjadual','Ujian Makmal','Ukuran',
+        'Varnishing',
+        'WAN','Wayar Elektrik Dan Aksesori','Wireless'
+    );
+
+    /**
+     * Return a random company name
+     * 
+     * @example 'AirAsia'
+     */
+    public static function companyName()
+    {
+        return static::randomElement(static::$companies);
+    }
+
+    /**
+     * Return a random industry
+     * 
+     * @example 'Automobil'
+     */
+    public static function industry()
```

Comment:
what's an industry?

Reply:
Category or area of business. Is any changes necessary here?